omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -1186,7 +1186,7 @@
 
                 var formattedDate = changeFieldDateToJavascriptDate(selectedDate);
 
-                <includeIf velocityTest="$fn.globalProperty('ugandaemr.maximumPatientsPerDay') != ''">
+                <includeIf velocityTest="$fn.globalProperty('ugandaemr.enableMaximumPatientsPerDay') == true">
```

Comment:
@slubwama Change this to a string to string comparison, not string to boolean comparison as it is currently